recipe/meta.yaml

```diff
@@ -22,12 +22,12 @@ requirements:
     - python
     - poetry-core 1.5
   run:
-    - markdown-it-py >=2.2.0,<3.0.0
-    - pygments >=2.13.0,<3.0.0
+    - markdown-it-py >=2.2.0
+    - pygments >=2.13.0
     - python
-    - typing_extensions >=4.0.0,<5.0.0  # [py<39]
+    - typing_extensions >=4.0.0  # [py<39]
```

Comment:
The upper bound still exists https://github.com/Textualize/rich/blob/v13.7.1/pyproject.toml#L31

Reply:
same as https://github.com/AnacondaRecipes/rich-feedstock/pull/7#discussion_r1665396348